source/gloperate-glkernel/include/gloperate-glkernel/stages/MultiFrameAggregationPipeline.h

```diff
@@ -40,10 +48,10 @@ class GLOPERATE_API MultiFrameAggregationPipeline : public Pipeline
 
 public:
     // Interfaces
-    RenderInterface renderInterface; ///< Interface for rendering into a viewer
+    gloperate::RenderInterface renderInterface; ///< Interface for rendering into a viewer
 
     // Inputs
-    Input<int> multiFrameCount; ///< Maximum number of frames to aggregate
+    gloperate::Input<int> multiFrameCount; ///< Maximum number of frames to aggregate
```

Comment:
We designed the ```gloperate::Stage``` in a way so that each subclass don't need to specify the gloperate namespace when using ```Input```s and ```Output```s. The former code should compile, too.